packages/eslint-plugin-lit-a11y/lib/rules/no-onchange.js

```diff
@@ -4,18 +4,21 @@
  */
 
 const { TemplateAnalyzer } = require('../../template-analyzer/template-analyzer.js');
+const { isHtmlTaggedTemplate } = require('../utils/isLitHtmlTemplate.js');
 
 //------------------------------------------------------------------------------
 // Rule Definition
 //------------------------------------------------------------------------------
 
 const applicableTypes = ['select', 'option'];
 
-module.exports = {
+/** @type {import("eslint").Rule.RuleModule} */
+const NoOnchangeRule = {
   meta: {
+    type: 'suggestion',
     docs: {
-      description: 'Enforce usage of onBlur over onChange for accessibility.',
-      category: 'Fill me in',
+      description: 'Enforce usage of @blur over @change for accessibility.',
```

Comment:
Unrelated to this PR, should the error message suggest using `@input` instead of `@change` if user wants to track every user input? That's probably a common mistake, but swapping `@input` for `@blur` won't make sense to users who mistook `@change` for `@input`